src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -234,6 +183,18 @@
             </td>
           </tr>
          }
+          <tr>
+            <td class="bold">
+              Placement Target
+              <cd-helper class="text-pre-wrap">
+                <span i18n>
+                  Placement Target defines the destination and rules for moving objects between
+                  storage tiers.
```

Comment:
Question from my side: is this right? From what I can see in the docs: https://docs.ceph.com/en/latest/radosgw/placement/#placement-targets

> Placement targets control which [Pools](https://docs.ceph.com/en/latest/radosgw/pools) are associated with a particular bucket.